auth0.gemspec

```diff
@@ -1,22 +1,22 @@
 # -*- encoding: utf-8 -*-
-$:.push File.expand_path("../lib", __FILE__)
-require "auth0/version"
+$LOAD_PATH.push File.expand_path('../lib', __FILE__)
```

Comment:
Yes, is a rule with autofix, is the same use $: or $LOAD_PATH. 
An example in [Stack Overflow](http://stackoverflow.com/a/6671578/2839987)